src/components/AppAlert.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div
+    class="p-4 mb-5 "
+    :class=" [
+      'bg-' + alertType.color + '-50 border-' + alertType.color + '-200 ',
```

Comment:
Don't concatenate Tailwind classes.
PurgeCSS will remove the classes that couldn't be found by a simple regex.

See:
- https://tailwindcss.com/docs/optimizing-for-production
- https://markus.oberlehner.net/blog/reusable-functional-vue-components-with-tailwind-css/